proposals/TAG review request.md

```diff
@@ -11,18 +11,25 @@ I'm requesting a TAG review of:
 Further details (optional):
 
   - Relevant time constraints or deadlines: 
-  * TAG review feedback expected by April 5th
-  * CR transition April 9th
+     * TAG review feedback expected by April 5th
+     * CR transition planned for April 9th
+        - in order to make June 25 deadline for REC
+        - end of WG charter is June 30
   
-  Additional schedule: https://www.w3.org/WoT/IG/wiki/Main_WoT_WebConf
+  [Additional schedule details and constraints](https://www.w3.org/WoT/IG/wiki/Main_WoT_WebConf)
   
   
-  - [ ] I have read and filled out the [Self-Review Questionnare on Security and Privacy](https://www.w3.org/TR/security-privacy-questionnaire/). The [assessment is here](url).
+  - [X] I have read and filled out the [Self-Review Questionnare on Security and Privacy](https://www.w3.org/TR/security-privacy-questionnaire/). The [assessment is here](https://github.com/w3c/wot-architecture/blob/master/proposals/security_and_privacy.md).
   - [X] I have reviewed the TAG's [API Design Principles](https://w3ctag.github.io/design-principles/)
 
 You should also know that...
 
-[please tell us anything you think is relevant to this review]
+- The [WoT Security and Privacy Considerations](https://github.com/w3c/wot-security/) document provides extra discussion of security with an IoT focus.
+- The related [WoT Thing Description](https://github.com/w3c/wot-thing-description) document is being submitted for TAG review at the same time.
+    - Tests of implementations will be associated with this document
+- The [WoT Binding Templates](https://github.com/w3c/wot-binding-templates) non-REC track informative document describes an optional WoT building block.
+- The [WoT Scripting API](https://github.com/w3c/wot-binding-templates) non-REC track informative document describes an optional WoT building block.
```

Comment:
is there a better term for "non-REC track informative document"? 
e.g. informative WG note?

Reply:
Sure, I can change that so it's positive rather than negative.  BTW at some point I will revise the TD note as well and will make them consistent.

Reply:
Changed in both occurrences to "informative WG note describing an optional WoT building block."